Content.Server/Radio/EntitySystems/RadioDeviceSystem.cs

```diff
@@ -95,6 +97,34 @@ public void ToggleRadioMicrophone(EntityUid uid, EntityUid user, bool quiet = fa
             RemCompDeferred<ActiveListenerComponent>(uid);
     }
 
+    private void OnAlternativeVerb(EntityUid uid, RadioMicrophoneComponent component, GetVerbsEvent<AlternativeVerb> args)
+    {
```

Comment:
If feels a bit weird to make this an alt-click verb unless you somehow set it up so that some resonable default frequency (e.g.. common) always has the highset priority & still enabled even when selected. Otherwise the result of alt-clicking is just unpredictable?

So maybe just make this a generic verb-verb